tests/unit/test_branch_self_parallel_tools.py

```
@@ -196,72 +88,48 @@ def add_reply_side_effect(*args, **kwargs):
-        ]
+        original_node.content = "Regular message"
+        original_node.tool_calls = None  # No tool calls
         original_node.replies = []
 
-        # Mock the dummy node that will be created
+        # Mock the dummy node
         dummy_node = Mock(spec=ConversationNode)
         dummy_node.replies = []
 
-        # Setup _add_reply to add dummy_node to replies and return it
         def add_reply_side_effect(*args, **kwargs):
```

Ruff (0.14.13)

Unused function argument: `args`

(ARG001)